app/assets/stylesheets/application.scss

```diff
@@ -119,107 +134,175 @@
     width: 13rem;
     margin-left: -.599rem;
     border-color: $base;
+
+    &:hover {
+      background-color: $third-color;
+      color: $oil;
+      transition: all .25 ease-in-out;
+    }
   }
 }
 
-.welcome {
+section.welcome p {
   font-family: sans-serif;
-  width: 80%;
-  text-align: left;
-  margin-left: 10%;
-  margin-right: 10%;
+  text-align: center;
+  font-size: 1.4rem;
+  margin-left: 0;
+  margin-right: 0;
   align-self: center;
-  style {
-    color: $charcoal;
-  }
 
   i {
     color: $charcoal;
   }
+
+  strong {
+    font-weight: bolder;
+    color:$secondary-color;
+  }
+
+  span {
+    color: $third-color;
+    font-weight: bolder;
+  }
 }
 
 // bucket-list index page
 
+.display-groups {
+  margin: 0;
+  padding: 0;
+}
+
 .grid-groups li {
   @include block-grid(
-    $per-row: 5,
+    $per-row: 4,
     $spacing: $block-grid-default-spacing,
     $base-style: true
   );
 
-  margin: 1%;
+  margin: 0;
+  padding: 1%;
 }
 
-div.wrap-title-and-description {
+div.wrap-top {
+  display: inline-block;
+  width: 100%;
   padding: 1.5rem 2rem .5rem 2rem;
   margin: 0;
   background-color: rgba(250,250,250,0.7);
+  height: 250px;
 
   h4 {
-    font-weight: bolder;
-    color: $secondary-color;
+    font-weight: bold;
+    padding: 0;
+    margin: 0;
   }
 
-  p.description {
-    margin: 0;
+  h4 a {
+    padding: .5rem 0rem .5rem 0rem;
   }
+
 }
 
-div.wrap-title-and-description:hover a {
-  display: inline-block;
-  width: 100%;
+p#description.description.bucket-list-bloc {
+  text-align: left;
+  color: $charcoal;
+  font-style: italic;
+  padding: 1rem 0rem 0rem 0rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `1rem 0rem 0rem` instead of `1rem 0rem 0rem 0rem`<br>`0rem` should be written without units as `0`